src/views/product-tutorials-view/components/product-view-content/components/collections-stack/types.ts

```diff
@@ -5,12 +5,12 @@ import {
 
 export interface CollectionsStackProps {
   /** Heading to show above the collection cards. */
-  heading: string
+  heading?: string
   /**
    * Identifier for the heading, which should unique in the context of the page.
    * Note: headingSlug is added after fetching content from the Learn API
    */
-  headingSlug: string
+  headingSlug?: string
```

Comment:
[`heading` is not required for `CollectionsStack` blocks](https://github.com/hashicorp/dev-portal/blob/8e1f3770c6e361a38b27f4556e0683b3d6ada801/src/lib/learn-client/schemas.ts#L57), so these should in fact be optional props.